src/app-router.ts

```diff
@@ -1,10 +1,19 @@
 import * as LogManager from 'aurelia-logging';
-import {Container} from 'aurelia-dependency-injection';
-import {History} from 'aurelia-history';
-import {Router} from './router';
-import {PipelineProvider} from './pipeline-provider';
-import {isNavigationCommand} from './navigation-commands';
-import {EventAggregator} from 'aurelia-event-aggregator';
+import { Container } from 'aurelia-dependency-injection';
+import { History, NavigationOptions } from 'aurelia-history';
+import { Router } from './router';
+import { PipelineProvider } from './pipeline-provider';
+import { isNavigationCommand } from './navigation-commands';
+import { EventAggregator } from 'aurelia-event-aggregator';
+import { NavigationInstruction } from '.';
```

Comment:
./navigation-instruction?

Reply:
they was all rexported from index, I think that's why I got this automatically completed from the wrong module